test/letters/containers/AddressSection.unit.spec.jsx

```diff
@@ -28,37 +27,62 @@ const defaultProps = {
 
 describe('<AddressSection>', () => {
   it('should render', () => {
-    const tree = SkinDeep.shallowRender(<AddressSection {...defaultProps}/>);
+    const tree = SkinDeep.shallowRender(<AddressSection { ...defaultProps }/>);
```

Comment:
Oh weird...And the linter doesn't care about this? Previously, we always had to make sure we had no spaces between our `{}`s inside jxs tags. Huh. :man_shrugging: